components/sidebar/header/dropdown/sidebar_header_dropdown.jsx

```diff
@@ -254,18 +254,23 @@ export default class SidebarHeaderDropdown extends React.Component {
                     teamId={this.props.teamId}
                     permissions={[Permissions.INVITE_USER]}
                 >
-                    <li>
-                        <button
-                            className='style--none'
-                            id='sendEmailInvite'
-                            onClick={this.showInviteMemberModal}
-                        >
-                            <FormattedMessage
-                                id='navbar_dropdown.inviteMember'
-                                defaultMessage='Send Email Invite'
-                            />
-                        </button>
-                    </li>
+                    <TeamPermissionGate
+                        teamId={this.props.teamId}
+                        permissions={[Permissions.ADD_USER_TO_TEAM]}
+                    >
+                        <li>
+                            <button
+                                className='style--none'
+                                id='sendEmailInvite'
+                                onClick={this.showInviteMemberModal}
```

Comment:
Not blocking but maybe we should start on using the `ToggleModalButtonRedux` component which under the hood is using a redux-style action in opening a modal. `GlobalActions.showGetTeamInviteLinkModal` is still in the old flux-like architecture.

Reply:
This PR goes to the feature branch advanced-permissions-phase-1, so I think is better to leave that change out of this PR.